ONSdigital/census-rh-ui
# Cr 1662 error writeback

# Motivation and Context
Feeds back valid values when inputs are missed for 'Web Chat' and 'Enter Name'

# What has changed
Updates to flash messaging for web chat and enter name errors to send back inputted field values
Updates to output templates to write in valid values to non-erroring fields

No Cucumber updates required

# How to test?
In Web Chat and Enter Name step in fulfilment journeys complete some but not all fields and submit
Valid values should be maintained, whilst empty values should error

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1662
